ImportError: cannot import name 'get_current_traceback' from 'werkzeug.debug.tbtools'

Running into a similar issue as in plotly/dash#1992:

```
Traceback:
/usr/local/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
vizx_dashboards/__init__.py:26: in <module>
    from .create_dashboard import main as create_dashboard  # noqa: F401
vizx_dashboards/create_dashboard.py:22: in <module>
    from jupyter_dash import JupyterDash
../../../brix_venv/vizx_dashboards/lib/python3.7/site-packages/jupyter_dash/__init__.py:2: in <module>
    from .jupyter_app import JupyterDash
../../../brix_venv/vizx_dashboards/lib/python3.7/site-packages/jupyter_dash/jupyter_app.py:21: in <module>
    from werkzeug.debug.tbtools import get_current_traceback
E   ImportError: cannot import name 'get_current_traceback' from 'werkzeug.debug.tbtools' (/home/********/brix_venv/vizx_dashboards/lib/python3.7/site-packages/werkzeug/debug/tbtools.py)
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 All notable changes to `jupyter-dash` will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Fixed
+  - Fixed `werkzeug` 2.1.0 import and `skip` calculation
+
 ## 0.4.1 - 2022-02-16
 ### Fixed
  - Support Dash 2.1, fix `AttributeError: Read-only... requests_pathname_prefix`

--- a/jupyter_dash/jupyter_app.py
+++ b/jupyter_dash/jupyter_app.py
@@ -9,6 +9,7 @@ import io
 import re
 import sys
 import inspect
+import traceback
 import warnings
 
 from IPython import get_ipython
@@ -20,27 +21,12 @@ import uuid
 from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
 
 
-def _get_skip(error: Exception, divider=2):
-
-    try:
-        # pylint: disable=import-outside-toplevel
-        from werkzeug.debug import tbtools
-    except ImportError:
-        tbtools = None
-
-    # werkzeug<2.1.0
-    if hasattr(tbtools, "get_current_traceback"):
-        tb = tbtools.get_current_traceback()
-        text = tb.plaintext.splitlines()
-
-    if hasattr(tbtools, "DebugTraceback"):
-        tb = tbtools.DebugTraceback(error)  # pylint: disable=no-member
-        text = tb.render_traceback_text().splitlines()
-    
+def _get_skip(error: Exception):
+    tb = traceback.format_exception(type(error), error, error.__traceback__)
     skip = 0
     for i, line in enumerate(text):
         if "%% callback invoked %%" in line:
-            skip = int((i + 1) / divider)
+            skip = i + 1
             break
     return skip
 

--- a/jupyter_dash/jupyter_app.py
+++ b/jupyter_dash/jupyter_app.py
@@ -17,10 +17,32 @@ from IPython.core.ultratb import FormattedTB
 from ansi2html import Ansi2HTMLConverter
 import uuid
 
-
-from werkzeug.debug.tbtools import get_current_traceback
-
 from .comms import _dash_comm, _jupyter_config, _request_jupyter_config
+
+
+def _get_skip(error: Exception, divider=2):
+
+    try:
+        # pylint: disable=import-outside-toplevel
+        from werkzeug.debug import tbtools
+    except ImportError:
+        tbtools = None
+
+    # werkzeug<2.1.0
+    if hasattr(tbtools, "get_current_traceback"):
+        tb = tbtools.get_current_traceback()
+        text = tb.plaintext.splitlines()
+
+    if hasattr(tbtools, "DebugTraceback"):
+        tb = tbtools.DebugTraceback(error)  # pylint: disable=no-member
+        text = tb.render_traceback_text().splitlines()
+    
+    skip = 0
+    for i, line in enumerate(text):
+        if "%% callback invoked %%" in line:
+            skip = int((i + 1) / divider)
+            break
+    return skip
 
 
 class JupyterDash(dash.Dash):
@@ -356,18 +378,12 @@ class JupyterDash(dash.Dash):
     ):
 
         @self.server.errorhandler(Exception)
-        def _wrap_errors(_):
+        def _wrap_errors(error):
             """Install traceback handling for callbacks"""
             self._traceback = sys.exc_info()[2]
 
             # Compute number of stack frames to skip to get down to callback
-            tb_werkzeug = get_current_traceback()
-            skip = 0
-            if dev_tools_prune_errors:
-                for i, line in enumerate(tb_werkzeug.plaintext.splitlines()):
-                    if "%% callback invoked %%" in line:
-                        skip = int((i + 1) / 2)
-                        break
+            skip = _get_skip(error) if dev_tools_prune_errors else 0
 
             # Customized formatargvalues function so we can place function parameters
             # on separate lines

